README.md

```diff
@@ -14,6 +14,10 @@ American Water Resources Association (JAWRA)* 1-15, DOI: 10.1111/1752-1688.12434
 
 Snow, Alan Dee, "A New Global Forecasting Model to Produce High-Resolution Stream Forecasts" (2015). All Theses and Dissertations. Paper 5272. http://scholarsarchive.byu.edu/etd/5272
 
+<<<<<<< HEAD
```

Comment:
These tags shouldn't be there. I think the conflict fix didn't work. Can you try again?